docs/sources/developers/plugins/create-a-grafana-plugin/extend-a-plugin/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Extend a Grafana plugin
+menuTitle: Extend a plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - development
+  - extension
+  - documentation
+description: An index of how-to topics for extending or enhancing Grafana plugins.
+weight: 200
+---
+
+# Extend a Grafana plugin
+
+This section contains how-to topics for extending or enhancing Grafana plugins.
+
+- [Add annotations]({{< relref "./add-support-for-annotations.md" >}})
+- [Add anonymous usage reporting]({{< relref "./add-anonymous-usage-reporting.md" >}})
+- [Add authentication for a data source plugin]({{< relref "./add-authentication-for-data-source-plugins.md" >}})
+- [Add distributed tracing for backend plugins]({{< relref "./add-distributed-tracing-for-backend-plugins.md" >}})
+- [Add Explore queries]({{< relref "./add-support-for-explore-queries.md" >}})
```

Comment:
In the ToC this is still Add features to Explore queries, suggest updating here to match or adding menutitle to change in the other file